packages/client/src/views/SysAdmin/Config/Forms/FormConfigSettings.tsx

```diff
@@ -0,0 +1,335 @@
+}
+
+export type IFormConfigSettingsProps = IntlShapeProps & {
+  applicationConfig: IOfflineData
+} & DispatchProps
+interface State {
+  modalName: string
+  introductionPage: boolean
+  numberOfAddresses: number
+  showModal: boolean
+  errorOccured: boolean
+  showNotification: boolean
+  notificationMessages: string
+}
+
+enum ConfigModals {
+  INTRODUCTION_PAGE = 'changeIntroductionPage',
+  ADDRESSES = 'changeAddresses'
+}
+
+class FormConfigSettingsComponent extends React.Component<
+  IFormConfigSettingsProps,
+  State
+> {
+  constructor(props: IFormConfigSettingsProps) {
+    super(props)
+    this.state = {
+      modalName: EMPTY_STRING,
+      introductionPage:
+        this.props.applicationConfig.config.HIDE_EVENT_REGISTER_INFORMATION,
+      numberOfAddresses: this.props.applicationConfig.config.ADDRESSES,
+      showModal: false,
+      errorOccured: false,
+      showNotification: false,
+      notificationMessages: EMPTY_STRING
+    }
+  }
+
+  changeValue = async () => {
+    if (this.state.modalName === ConfigModals.INTRODUCTION_PAGE) {
+      try {
+        await callUpdateHideEventRegisterInformationMutation(
+          this.state.introductionPage,
+          this.props
+        )
+        this.toggleConfigModal()
+        this.setState({
+          showNotification: true,
+          notificationMessages: this.props.intl.formatMessage(
+            messages.introductionPageSuccessNotification,
+            {
+              action: this.state.introductionPage
+                ? this.props.intl.formatMessage(messages.enable)
+                : this.props.intl.formatMessage(messages.disable)
+            }
+          )
+        })
+      } catch {
+        this.setState({
+          errorOccured: true
+        })
+      }
+    } else if (this.state.modalName === ConfigModals.ADDRESSES) {
+      try {
+        await callUpdateAddressesMutation(
+          this.state.numberOfAddresses,
+          this.props
+        )
+        this.toggleConfigModal()
+        this.setState({
+          showNotification: true,
+          notificationMessages: this.props.intl.formatMessage(
+            messages.noOfAddressesSuccessNotification
+          )
+        })
+      } catch {
+        this.setState({
+          errorOccured: true
+        })
+      }
+    }
+  }
+
+  toggleOnChange = () => {
+    this.setState({ introductionPage: !this.state.introductionPage })
+  }
+
+  setNumberOfAddresses = (noOfAddresses: string) => {
+    this.setState({ numberOfAddresses: parseInt(noOfAddresses) })
+  }
+
+  toggleConfigModal = () => {
+    this.setState({ showModal: !this.state.showModal })
+  }
+
+  render() {
+    const { intl } = this.props
+    const { showModal, modalName, introductionPage, numberOfAddresses } =
+      this.state
+
+    return (
+      <SysAdminContentWrapper>
+        <Content
+          title={intl.formatMessage(messages.settingsTitle)}
+          titleColor={'copy'}
+        >
+          <ListView
```

Comment:
Shouldn't we use ListViewSimplified

Reply:
yes

Reply:
@atiqzaman-dsi  @jpye-finch  I have updated the component with ListViewSimplified.

And @Zangetsu101 will update the  SideNav and TopNav wrapper for form config.